File: Minimum Steps (Array Series #6).dart
int minimumSteps(List<int> nums, int value) {
  nums.sort();
  if (nums[0] > value) return 0;
   
  int i = 0;
  int t = 0;
  
  while(t < value) t += nums[i++];
  
  return i - 1;
}

File: Minimum Steps (Array Series #6).ts
export function minimumSteps(nums: number[], value: number) {
  nums.sort((a,b) => a-b)
  let count = 0;
  let sum = nums[0];
  let index = 1;
  while(sum<value){
    sum+=nums[index]
    index+=1
    count+=1
  }
  return count;
}

